components/form/TaskForm.tsx

```diff
@@ -2,7 +2,7 @@ import { UserContext } from "@/lib/contexts/FirebaseAuthContext"
 import { useUpdateTasks } from "@/lib/hooks/useTasks"
 import { Project, Task } from "@/lib/types/apiTypes"
 import { FormBase } from "@/lib/types/forms"
-import { Box, Checkbox, Input } from "@chakra-ui/react"
+import { Box, Checkbox as CheckBoxField, Input } from "@chakra-ui/react"
```

Comment:
We don't want this `as`-statement here. What happens now is that on line 85 we continue to render `Checkbox` but now it's simply called `CheckBoxField`. You'll need to drop `Checkbox` from the imports completely and the "auto-fix" the imports on line 85 on VSCode.